ui/src/ensembler/components/table/ListEnsemblingJobsForEnsemblerTable.js

```diff
@@ -0,0 +1,120 @@
+import React, { Fragment, useEffect, useState } from "react";
+import {
+  EuiCallOut,
+  EuiBasicTable,
+  EuiText,
+} from "@elastic/eui";
+import { DeploymentStatusHealth } from "../../../components/status_health/DeploymentStatusHealth";
+import { JobStatus } from "../../../services/job/JobStatus";
+import { useConfig } from "../../../config";
+import { useTuringApi } from "../../../hooks/useTuringApi";
+
+
+export const ListEnsemblingJobsForEnsemblerTable = ({
+  projectID,
+  ensemblerID,
+  setCanDeleteEnsembler,
+  canDeleteEnsembler
+}) => {
+  const [results, setResults] = useState({ inactiveItems: [], activeItems:[], totalInactiveCount: 0, totalActiveCount:0 });
+
+  const {
+    appConfig: {
+      tables: { defaultTextSize },
+    },
+  } = useConfig();
+
+  const [{ data, isLoaded, error }] = useTuringApi(
+    `/projects/${projectID}/jobs?ensembler_id=${ensemblerID}`,
```

Comment:
Since this API is paginated, we need to call it in a loop, one page at a time, right?

Reply:
Since the dialog doesnt implement pagination, it will have infinite scroll, so we dont need to set pagination for the api call

Reply:
> Since the dialog doesnt implement pagination, it will have infinite scroll, so we dont need to set pagination for the api call

I think Krithika's referring to a concern whereby the API returns the list of ensemblers in a paginated response by default (e.g. with the default page length set as 10, and only the first page gets returned by default), in which case what we normally do in the UI was to use some nifty trick to issue repeated API calls to get the entire list of _something_, if the response is paginated by default.

That's the case for XP, where the API response to get a list of experiments, for example is paginated by default if we are using neither field selectors (e.g. `id`, `name`, etc. to get specific fields only in the response) nor pagination options.

However, it seems like the way Turing's list ensemblers API endpoint works differently - there's not setting of any default page length if it's not specified anywhere (i.e. the entire list gets returned in the first page, see [this](https://github.com/caraml-dev/turing/blob/main/api/turing/service/pagination.go#L38)). So in this case I don't think there's a need to loop through the API responses in this case.